packages/lodestar/test/unit/eth1/impl/ethers.test.ts

```diff
@@ -120,7 +121,8 @@ describe("Eth1Notifier", () => {
       logger,
     });
     await expect(
-      eth1.start()
+      // @ts-ignore
```

Comment:
why?

Reply:
`eth1.start()` does not await the `startProcessEth1Blocks` which is private so I have to do like this. The other way is to publish `startProcessEth1Blocks` just for test. Any ideas?

Reply:
ah, either leave like this or introduce some events like "eth1BlockProcessStarted". I'm fine either way :slightly_smiling_face: